### PR TITLE
add contact list creator

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

**Comment by tentacode** [comment]
tu peux mettre la doc qui va avec la synchro ? dur de dire si ça répond au besoin sans voir la doc

**Comment by gignonje** [comment]
👍

**Comment by nicolasThal** [comment]
@tentacode I have added some documentation on the subscription/unsubscription usage

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "require": {
         "php": ">=5.5",
         "symfony/swiftmailer-bundle": "~2.3",
-        "mailjet/mailjet-apiv3-php": "~1.1"
+        "mailjet/mailjet-apiv3-php": "~1.1",
+        "doctrine/orm": "^2.4.8"
```
Comment (yoye) [inline composer.json:15]: C'est quoi qui nécessite cette version de doctrine/orm ?

Comment (nicolasThal) [inline composer.json:15]: I have based my requirement based on the one in the adback project. I haven't looked to get a older version.

We could check an other one if you want

**Pull request merged**